src/services/utilServices/Sanitizer.ts

```diff
@@ -1,5 +1,19 @@
 import DOMPurify from "isomorphic-dompurify"
 
+/**
+ * While we do have a CSP in place, we want to further restrict the content that
+ * that the user can input.
+ * NOTE: Any changes to this list should also be updated in frontend code
+ */
+const ALLOWED_SRC = [
+  "//www.instagram.com/embed.js",
+  "https://www.evvochannel.tv/jwplayer7/jwplayer.js",
+  "/jquery/resize-tables.js",
+  "/jquery/jquery.min.js",
+  "/jquery/bp-menu-new-tab.js",
+  "//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js",
+]
```

Comment:
question (blocking): where did you get `evvochannel.tv` and `s3.amazonaws.com` ones from? Those domains are not whitelisted inside our CSP currently.

Reply:
wah interesting, i did a grep search on existing repos to maintain parity, which is where i got this. Will remove these two + let ops know that this functionality for these sites are broken